Add Long running query metric

AnishTiwari opened the pull request:
Added long running query metric to the common metric api

Close #3

AnishTiwari committed:
Add Long running query metric
Added long running query metric to the common metric api

lakshmi-udaan reviewed (changes requested):
Minor changes requested. Rest LGTM.

AnishTiwari committed:
Fix Removed isParameterized flag
Removed the flag for now, will raise an issue to handle the flag in an appropriate manner
